tests/integration/api/users/UpdateTest.php

```diff
@@ -26,7 +27,24 @@ protected function setUp(): void
         $this->prepareDatabase([
             'users' => [
                 $this->normalUser(),
-            ]
+                [
```

Comment:
We decided to put this here and not in `RetrievesAuthorizedUsers` since it's only used in one test case